code/test/e2e/loggedin/filter_wizard.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test, expect } from '@playwright/test';
+
+async function delay(ms = 1000) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+test('Additional filter wizard', async ({ page }, { config }) => {
+  const { baseURL } = config.projects[0].use;
+  await page.goto(baseURL + '/ui/welcome');
+  await page.getByRole('link', { name: 'edges' }).click();
+  await page.pause();
+  await page.getByRole('button', { name: 'Filter' }).click();
+  await page.getByText('attribute name').click();
+  await page.getByRole('option', { name: 'name' }).click();
+  await page.locator('div[role="alert"]:has-text("operation")').click();
+  await page.getByRole('option', { name: 'Equal' }).getByText('Equal').click();
+  await page.getByText('value').click();
+  await page.locator('span:has-text("e2e-Test-Do_not_delete")').click();
+  await page.getByRole('button', { name: 'Done' }).click();
+  await page.waitForURL('https://nui.localhost/ui/edges?view=table&nuvlabox=name=%27e2e-Test-Do_not_delete%27');
```

Comment:
e2e test failing and hardcoded urls

Reply:
Will send a video with bugs:
- popup visible when modal is open
- remove filter doesn't work properly on edge page

Reply:
Fixed both plus the e2e test URLs